PATCH-P0-FEATURE-SUSE: Moving SafeProcess to /usr/bin

We prefer to have this script somewhere else, so we moved it out of
the mysql-test directory.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: mysql-test/lib/My/SafeProcess.pm
===================================================================
--- mysql-test/lib/My/SafeProcess.pm.orig
+++ mysql-test/lib/My/SafeProcess.pm
@@ -102,7 +102,7 @@ sub find_bin {
   if (IS_WIN32PERL or IS_CYGWIN)
   {
     # Use my_safe_process.exe
-    my $exe= my_find_bin($bindir, ["lib/My/SafeProcess", "My/SafeProcess"],
+    my $exe= my_find_bin($bindir, ["../../bin", "lib/My/SafeProcess", "My/SafeProcess"],
 			 "my_safe_process");
     push(@safe_process_cmd, $exe);
 
@@ -112,7 +112,7 @@ sub find_bin {
   else
   {
     # Use my_safe_process
-    my $exe= my_find_bin($bindir, ["lib/My/SafeProcess", "My/SafeProcess"],
+    my $exe= my_find_bin($bindir, ["../../bin", "lib/My/SafeProcess", "My/SafeProcess"],
 			 "my_safe_process");
     push(@safe_process_cmd, $exe);
   }
